pareto/tests/test_utilities.py

```diff
@@ -0,0 +1,337 @@
+#####################################################################################################
+# PARETO was produced under the DOE Produced Water Application for Beneficial Reuse Environmental
+# Impact and Treatment Optimization (PARETO), and is copyright (c) 2021-2023 by the software owners:
+# The Regents of the University of California, through Lawrence Berkeley National Laboratory, et al.
+# All rights reserved.
+#
+# NOTICE. This Software was developed under funding from the U.S. Department of Energy and the U.S.
+# Government consequently retains certain rights. As such, the U.S. Government has been granted for
+# itself and others acting on its behalf a paid-up, nonexclusive, irrevocable, worldwide license in
+# the Software to reproduce, distribute copies to the public, prepare derivative works, and perform
+# publicly and display publicly, and to permit others to do so.
+#####################################################################################################
+"""
+Test utilities
+"""
+
+from pareto.strategic_water_management.strategic_produced_water_optimization import (
+    WaterQuality,
+    create_model,
+    Objectives,
+    solve_model,
+    PipelineCost,
+    PipelineCapacity,
+    RemovalEfficiencyMethod,
+)
+from pareto.utilities.get_data import get_data
+from pareto.utilities.results import (
+    generate_report,
+    PrintValues,
+    OutputUnits,
+    is_feasible,
+    nostdout,
+)
```

Comment:
```suggestion
```
These imports are all unused and can be removed.

Reply:
Addressed